osbrain/proxy.py

```diff
@@ -122,7 +122,7 @@ def wait_for_running(self, timeout=3.):
             The object itself.
         """
         time0 = time.time()
-        while not self.get_attr('running'):
+        while not self.unsafe.is_running():
```

Comment:
Maybe `.is_running()` should be unsafe by default. What do you think?

If we want to check the agent is running *and* responding, then we have the  `.ping()` method.

Reply:
It probably should. It's a bit weird to call a method to check if the agent is running and instead get an error telling you that you can't call methods on the agent because it's not running. That is what happened to me here, by the way, and the reason why I changed this invocation to `unsafe`.

I'll change it by default 👍